designer/client/src/components/spinner/Spinner.tsx

```diff
@@ -12,11 +12,11 @@ function LoaderSpinner(props: Props) {
 export default LoaderSpinner;
 
 function CircularProgressWrapper() {
-    const theme = useTheme();
```

Comment:
@Dzuming please verify this

Reply:
this 

`const theme = useTheme();`

is the correct one

Reply:
ok, fixed